src/Components/Components/test/ParameterViewTest.Assignment.cs

```diff
@@ -181,7 +181,7 @@ public void IncomingParameterMatchesPropertyNotDeclaredAsParameter_Throws()
         Assert.Equal(default, target.IntProp);
         Assert.Equal(
             $"Object of type '{typeof(HasPropertyWithoutParameterAttribute).FullName}' has a property matching the name '{nameof(HasPropertyWithoutParameterAttribute.IntProp)}', " +
-            $"but it does not have [{nameof(ParameterAttribute)}] or [{nameof(CascadingParameterAttribute)}] applied.",
+            $"but it does not have [{nameof(ParameterAttribute)}], [{nameof(CascadingParameterAttribute)}] or [{nameof(SupplyParameterFromFormAttribute)}] applied.",
```

Comment:
Same question about whether `[SupplyParameterFromForm]` can be used without `[Parameter]`.

Reply:
https://github.com/dotnet/aspnetcore/pull/48412#discussion_r1206490285

Reply:
The conclusion here is that we are going to have `[QueryParameter]` and `[FormParameter]` as cascading values, and that's the only thing you'll need to specify them.